lua/nvterm/terminal.lua

```diff
@@ -59,7 +59,11 @@ local ensure_and_send = function(cmd, type)
     end
   end
   local term = select_term()
-  a.nvim_chan_send(term.job_id, cmd .. "\n")
+  if vim.loop.os_uname().sysname == "Windows_NT" then
+    a.nvim_chan_send(term.job_id, cmd .. "\r")
```

Comment:
isn't on windows, the correct newline sequence is `\r\n`

Reply:
before pushing i tried executing and its working as expected